main/adapters/http_server/csr_fetcher.go

```diff
@@ -11,6 +11,9 @@ import (
 type GetCSR func(uid uuid.UUID) (csr []byte, err error)
 
 func FetchCSR(auth string, get GetCSR) http.HandlerFunc {
+	if len(auth) == 0 {
+		panic("missing auth token for CSR creation endpoint")
```

Comment:
yes, as this function returns only a function and no error. With this check I wanted to make sure that it would be impossible to initialize this endpoint without an auth token.